core/src/main/java/com/adobe/aem/commons/assetshare/components/details/impl/VideoImpl.java

```diff
@@ -30,27 +35,49 @@
         resourceType = { VideoImpl.RESOURCE_TYPE },
         defaultInjectionStrategy = DefaultInjectionStrategy.OPTIONAL)
 public class VideoImpl extends AbstractEmptyTextComponent implements Video {
```

Comment:
True - I'll hold off on this one for now though - most of the dupe is going to go once the legacy stuff is removed. 
The dupe is more of a "coincidence" anyhow, rather than planned.